knowledge-base/scheduler-display-allday.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Display Only All-Day Appointments in the Scheduler
+description: Learn how to customize your Scheduler for Blazor to exclusively display all-day events in a multiday timeline view, ensuring optimal scheduling clarity.
+type: how-to
+page_title: Display Only All-Day Appointments in the Scheduler
+slug: scheduler-display-allday
+tags: scheduler, timeline, all-day
+ticketid: 1576096
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Scheduler for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to display only all-day events in the Scheduler?
+* How to show only all-day appointments in a [Timeline View]({%slug scheduler-views-timeline%}) of the Scheduler?
+* How to create a multiday scheduler view that only displays the all-day events?
+
+## Solution
+
+To achieve the desired outcome, ensure that you apply in your configuration the following points:
```

Comment:
Try being more specific.
```suggestion
To display only all-day events in the Scheduler's Timeline View, include the following in your configuration:
```
You can even go with something as short as "To display only all-day events in the Scheduler's Timeline View:"